packages/libs/components/src/components/widgets/NumberAndDateInputs.tsx

```diff
@@ -202,6 +206,7 @@ function BaseInput({
       <div style={{ display: 'flex', flexDirection: 'row' }}>
         <TextField
           InputProps={{ classes }}
+          inputProps={{ step }}
```

Comment:
This is magic to me. How does passing a number in here affect the step? I would have expected to have to pass an object with "step: {step}" or something. 
Also MUI making an `InputProps` and an `inputProps` is not cool lol.

Reply:
>  Also MUI making an InputProps and an inputProps is not cool lol.

For real! Had to re-read the docs a couple times to figure this one out!

> This is magic to me. How does passing a number in here affect the step? I would have expected to have to pass an object with "step: {step}" or something.

This is using [object shorthand](https://attacomsian.com/blog/javascript-object-property-shorthand). Since `step` is both the name of the variable and the HTML attribute we're setting in the input, I went this route. In re-thinking this, I'm wondering if it's problematic that when `step` is undefined, we're passing in `{ step: undefined }`... 🤔

Reply:
ahhhhhhhhh object shorthand. Very cool! Oh JS, you surprise me in new ways every day!